classes/Kohana/Kostache/Layout.php

```diff
@@ -34,9 +34,13 @@ public function set_layout($layout)
 
 	public function render($class, $template = NULL)
 	{
+		if ($template == NULL)
```

Comment:
It will get passed to Kohana::find_file, and that treats the input as a string (with all consequences).

I'm basically manually patching Kostache for my app to properly work.